integration-tests/deployment/ccip/add_chain_test.go

```diff
@@ -51,14 +51,13 @@ func TestAddChainInbound(t *testing.T) {
 		},
 	)
 	err = DeployCCIPContracts(e.Env, e.Ab, DeployCCIPContractConfig{
-		HomeChainSel:       e.HomeChainSel,
-		FeedChainSel:       e.FeedChainSel,
-		ChainsToDeploy:     initialDeploy,
-		TokenConfig:        tokenConfig,
-		MCMSConfig:         NewTestMCMSConfig(t, e.Env),
-		FeeTokenContracts:  e.FeeTokenContracts,
-		CapabilityRegistry: state.Chains[e.HomeChainSel].CapabilityRegistry.Address(),
-		OCRSecrets:         deployment.XXXGenerateTestOCRSecrets(),
+		HomeChainSel:        e.HomeChainSel,
+		FeedChainSel:        e.FeedChainSel,
+		ChainsToDeploy:      initialDeploy,
+		TokenConfig:         tokenConfig,
+		MCMSConfig:          NewTestMCMSConfig(t, e.Env),
+		ExistingAddressBook: e.Ab,
```

Comment:
@AnieeG @0xAustinWang @dimkouv I simplified this by just passing the existing state then loading it inside